store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/integration/FederatedStoreRecursionIT.java

```diff
@@ -64,15 +63,14 @@
     private Graph proxyToRestServiceFederatedGraph;
 
     @Before
-    public void setUp() throws Exception {
+    public void _setup() throws Exception {
```

Comment:
Keep the method as `setUp` not `_setUp` graph isn't used in this class.
Override the `setUp` to do NOTHING.

```suggestion
    public void setUp() throws Exception {
```

Reply:
Done

Reply:
@m29827 Hi, you didn't keep this line as `setUp` like my comment requested.
and you didn't override this method to do NOTHING.

background: if you override 'setUp' in the particular class to do nothing then the graphs do not get added, so you then do not need to remove them in `_setUp`

Reply:
You're right, not sure how I missed that!